core/src/main/java/com/hcl/ecomm/core/services/impl/CreateCartServiceImpl.java

```diff
@@ -47,16 +47,23 @@ public String getEmptyCartPath() {
 	
 
 	@Override
-	public JSONObject createGuestCart() {
+	public JSONObject createCart(String customerToken) {
 		LOG.debug("createGuestCart method start.");
 		String scheme = "http";
+		String authToken = "";
+		String createGuestCartPath = "";
 		JSONObject createGuestCartRes = new JSONObject();
-		
 
 		try {
-			String authToken = loginService.getToken();
+			if(customerToken != null && !customerToken.isEmpty()) {
+				authToken = customerToken;
+				createGuestCartPath = config.customer_createCart_string();
+			}
+			else {
+				authToken = loginService.getToken();
+				createGuestCartPath = getEmptyCartPath();
+			}
 			String domainName = getDomainName();
-			String createGuestCartPath = getEmptyCartPath();
 			String url = scheme + "://" + domainName + createGuestCartPath;
 			LOG.info("createGuestCartPath  : " + url);
```

Comment:
@DeepaliAgrwl : change this to log, debug

Reply:
Done